### ISSUE TITLE
fireworks are not tradeable with the sign

### ISSUE THREAD

**Issue by Marchino1407** [EssentialsX/Essentials#5743]
### Type of bug

Other unexpected behaviour

### `/ess dump all` output

https://essentialsx.net/dump.html?id=c6f86574f922430b82b7523a9a0a80f2

### Bug description

fireworks are not trade with the sign because they do not enter the sign and I was wondering if it was somehow possible to use trade with them. Also because I tried every way with id but it still doesn't work

### Steps to reproduce

the bug cannot be activated because it is simply a trade error

### Expected behaviour

that you somehow manage to trade fireworks

### Actual behaviour

read above

**Comment by firework401** [comment]
yeah sure i've got the same problem when i try to insert the firework name

**Comment by pop4959** [comment]
I'm not quite sure what the bug is here. Would you be able to give an example of the type and contents of the sign you are interacting with as well as providing the error message you see. Thanks!

**Comment by JRoy** [comment]
Trade signs do not support items with custom nbt data. Closing as duplicate of #81
